components/inputs/test/input-text.visual-diff.js

```diff
@@ -1,96 +0,0 @@
-import { focusWithKeyboard, VisualDiff } from '@brightspace-ui/visual-diff';
-import puppeteer from 'puppeteer';
-
-describe('d2l-input-text', () => {
-
-	const visualDiff = new VisualDiff('input-text', import.meta.url);
-
-	let browser, page;
-
-	before(async() => {
-		browser = await puppeteer.launch();
-		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 3600 } });
-		await page.goto(`${visualDiff.getBaseUrl()}/components/inputs/test/input-text.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
-		await page.bringToFront();
-	});
-
-	after(async() => await browser.close());
-
-	['wc', 'sass'].forEach((type) => {
-		[
-			'basic',
-			'email',
-			'number',
-			'password',
-			'search',
-			'tel',
-			'url',
-			'disabled',
-			'placeholder',
-			'placeholder-disabled',
-			'invalid',
-			'invalid-disabled',
-			'aria-invalid',
-			'aria-invalid-disabled'
-		].forEach((name) => {
-			const id = `${type}-${name}`;
-			it(id, async function() {
-				const rect = await visualDiff.getRect(page, `#${id}`);
-				await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			});
-			if (name.indexOf('disabled') === -1) {
-				it(`${id}-focus`, async function() {
-					await focusWithKeyboard(page, `#${id}`);
-					const rect = await visualDiff.getRect(page, `#${id}`);
-					await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-				});
```

Comment:
Looks like we removed all these focus tests, that was intentional?

Reply:
So what I do (for my PRs and all the ones I'm reviewing) is check the files here: https://github.com/BrightspaceUI/core/pull/4110/files
And see if any are being removed instead of replaced, and verify I expected that

Reply:
Ah ok that makes sense, I'd just been relying on the report which is clearly not enough.

Why's there gotta be so many focus tests lol. I may remove some of these.

Reply:
Haha yeah I figured you were gonna say "Yup did it on purpose, they were basically all the same". But just wanted to check, I didn't look at the old pngs to see if they were useful

Reply:
Alright, I've updated things to keep a couple of the focus tests but remove the redundant ones.